src/components/Book.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Book = props => {
```

Comment:
**[optional]**
You could also handle it this way `const Book = ({ id, title, category }) => {`